repos/system_upgrade/common/models/dynamiclinker.py

```diff
@@ -0,0 +1,23 @@
+from leapp.models import fields, Model
+from leapp.topics import SystemFactsTopic
+
+
+class ConfigFile(Model):
+    topic = SystemFactsTopic
+
+    path = fields.String()
+    modified = fields.Boolean()
+
+
+class MainConfigFile(ConfigFile):
+    topic = SystemFactsTopic
+
+    modified_lines = fields.List(fields.String(), default=[])
```

Comment:
@PeterMocary These models names are generic too much. Even when it is not expected to produce them directly, I am afraid that it could cause some problems in future - e.g. if someone else creates model `ConfigFile`, the behaviour will be biased when importing classes to actors.. What about to use rather e.g. `LDConfigFile` and `LDMainConfigFile` instead? Also please add docstrings to classes and fields in case of models. To make it explicit when particular classes are not expected to be used as "messages" but only part of a specific msg, and make clear what each field is supposed to represent. We will use it in future for automatic generation of documentation for messages.

Reply:
Ugh wanted to add some comments but it totally slipped my mind. The naming concerns were addressed as well. :+1: